protocol/scripts/deploys/localhost/v2/999-localhost-fork-test.ts

```diff
@@ -0,0 +1,255 @@
+    await expect(await OWNER_V2_INSTANCES.TEMPLE_LINE_OF_CREDIT.INSTANCE.inRescueMode()).to.be.true;
+    await expect(
+      ALICE_V2_INSTANCES.TEMPLE_LINE_OF_CREDIT.INSTANCE.borrow(ethers.utils.parseEther('1'), await alice.getAddress()))
+      .to.be.revertedWithCustomError(ALICE_V2_INSTANCES.TEMPLE_LINE_OF_CREDIT.INSTANCE, "InvalidAccess"
+    );
+    await mine (OWNER_V2_INSTANCES.TEMPLE_LINE_OF_CREDIT.INSTANCE.setRescueMode(false));
+  }
+
+  /* Transfer ownership to multisig.
+   * The ownership assestemnts below only work after
+   * $workspace/protocol/scripts/deploys/mainnet/v2/post-deploy/999-transfer-ownership.ts
+   * has been executed.
+   */
+  {
+    // Core executor & rescuer accept their new roles on trv
+    await acceptOwnershipCore(TEMPLE_V2_ADDRESSES, TEMPLE_V2_INSTANCES.TREASURY_RESERVES_VAULT.INSTANCE);
+    
+    // Core executor & rescuer accept their new roles on tlc
+    await acceptOwnershipCore(TEMPLE_V2_ADDRESSES, TEMPLE_V2_INSTANCES.TEMPLE_LINE_OF_CREDIT.INSTANCE);
+    
+    // DSR strategy executor & rescuer accept their new roles
+    await acceptExecutorAndRescuer(TEMPLE_V2_INSTANCES.STRATEGIES.DSR_BASE_STRATEGY.INSTANCE, TEMPLE_V2_ADDRESSES.STRATEGIES.DSR_BASE_STRATEGY.EXECUTOR_MSIG, TEMPLE_V2_ADDRESSES.STRATEGIES.DSR_BASE_STRATEGY.RESCUER_MSIG);
+        
+    // Temple base strategy executor & rescuer accept their new roles
+    await acceptExecutorAndRescuer(TEMPLE_V2_INSTANCES.STRATEGIES.TEMPLE_BASE_STRATEGY.INSTANCE, TEMPLE_V2_ADDRESSES.STRATEGIES.TEMPLE_BASE_STRATEGY.EXECUTOR_MSIG, TEMPLE_V2_ADDRESSES.STRATEGIES.TEMPLE_BASE_STRATEGY.RESCUER_MSIG);
+    
+    // TLC strategy executor & rescuer accept their new roles
+    await acceptExecutorAndRescuer(TEMPLE_V2_INSTANCES.STRATEGIES.TLC_STRATEGY.INSTANCE, TEMPLE_V2_ADDRESSES.STRATEGIES.TLC_STRATEGY.EXECUTOR_MSIG, TEMPLE_V2_ADDRESSES.STRATEGIES.TLC_STRATEGY.RESCUER_MSIG);
+    
+    // Gnosis1 strategy executor & rescuer accept their new roles
+    await acceptExecutorAndRescuer(TEMPLE_V2_INSTANCES.STRATEGIES.GNOSIS_SAFE_STRATEGY1.INSTANCE, TEMPLE_V2_ADDRESSES.STRATEGIES.GNOSIS_SAFE_STRATEGY1.EXECUTOR_MSIG, TEMPLE_V2_ADDRESSES.STRATEGIES.GNOSIS_SAFE_STRATEGY1.RESCUER_MSIG);
+    
+    // Ramos strategy executor & rescuer accept their new roles
+    await acceptExecutorAndRescuer(TEMPLE_V2_INSTANCES.STRATEGIES.RAMOS_STRATEGY.INSTANCE, TEMPLE_V2_ADDRESSES.STRATEGIES.RAMOS_STRATEGY.EXECUTOR_MSIG, TEMPLE_V2_ADDRESSES.STRATEGIES.RAMOS_STRATEGY.RESCUER_MSIG);
+  }
+
+  /* Recovery gnosys1 strategy */
+  {
+    await mine(TEMPLE_V2_INSTANCES.STRATEGIES.GNOSIS_SAFE_STRATEGY1.INSTANCE.recoverToGnosis(
+      TEMPLE_V2_ADDRESSES.EXTERNAL.MAKER_DAO.DAI_TOKEN,
+      ethers.utils.parseEther("100")
+    ));
+
+  }
+
+  /* Add & Remove ramos liquidity */
+  { 
+    await addLiquidity(TEMPLE_V2_INSTANCES, ethers.utils.parseEther("10000"));
+    // await removeLiquidity(TEMPLE_V2_INSTANCES, ethers.utils.parseEther("2000"));
```

Comment:
got this test commented out because it's throwing me the following error, I'm thinking it probably is related to me using current mainnet pool, I believe I may need to deploy a new one instead:
```
ProviderError: Error: VM Exception while processing transaction: reverted with reason string 'SafeMath: subtraction overflow'
    at HttpProvider.request (/Users/anon/Documents/Web3/temple/protocol/node_modules/hardhat/src/internal/core/providers/http.ts:88:21)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async EthersProviderWrapper.send (/Users/anon/Documents/Web3/temple/protocol/node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)
```

Reply:
once you've updated from the other comments - let me know and we can step through this together. I haven't gone through this script in huge detail yet

Reply:
sorted out, I was using and incorrect contract address while deploying/setup ramos contract